static/js/compose_actions.js

```diff
@@ -62,19 +62,35 @@ function get_focus_area(msg_type, opts) {
     }
 
     if (msg_type === "stream") {
-        return "#stream_message_recipient_stream";
+        return "#id_compose_select_stream";
```

Comment:
I don't really know how to handle this and it feels a bit janky right now, `#id_compose_select_stream` is the textbox in the dropdown menu, but it's not actually used for anything here.

Essentially, for the stream case instead of focusing we want to click the dropdown instead of doing a focus. 

Is there a nicer way to represent that here?

Reply:
Do we want the dropdown to be closed or open when user clicks `new stream message`? I am sure I understand why you mean by you want it to be clicked and not focused?

Reply:
Hm, I'm not sure whether we want it to be open. Might be something to try out both ways and see what feels betters.

Reply:
> Soon we'll be using this same dropdown for PMs so we'll probably be removing get_focus_area anyways.

I posted on this here: https://github.com/zulip/zulip/pull/23355#discussion_r1080705508, but I think given this is called in one place, I think it's potentially reasonable to have this just return a string encoding of the "stream select element" as the answer for what should be focused, and have the caller execute "focusing that element" with something more complicated than calling `.focus()`, either via the focus handler on the input doing "open the dropup and focus it" or something else.

But I've not looked at what feels nice closely.